queries.sql

```diff
@@ -10,3 +10,63 @@ SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg >= '10.4' AND weight_kg <= '17.3';
 
 
+/* Add a column species of type string to animals table. */
+
+AlTER TABLE animals
+ADD COLUMN species char(100);
```

Comment:
- [ ] It would be better if we move lines 15,16 into the schema.sql file. Everything that creates or changes the table structure should be saved in the schema.sql file. In this case, we are changing the table structure

Reply:
Sure, thank you for your feedback.